include/deal.II/lac/sparse_vanka.templates.h

```diff
@@ -1,7 +1,7 @@
 // ------------------------------------------------------------------------
 //
 // SPDX-License-Identifier: LGPL-2.1-or-later
-// Copyright (C) 1999 - 2024 by the deal.II authors
+// Copyright (C) 1999-2024 by the deal.II authors
```

Comment:
Would you mind leaving this untouched? All of the other files use a style that has the spaces. Let's not make an exception here.

Reply:
Sorry, the spacing must have been deleted accidentally. I've fixed that now.